libosdp-sys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libosdp-sys"
-version = "3.0.6"
+version = "3.0.7"
```

Comment:
I reverted the version bump.

I think it is a good idea to keep `libosdp-sys` in sync with LibOSDP version numbers. That makes it way clearer what you are working with.
What I don't really know is if maybe `libosdp` should be versioned independently. If it has to stay in sync you cannot change it's public API without bumping LibOSDP's major version number, because Cargo expects SemVer compatible versioning and that means minor versions must not introduce breaking changes. So maybe it's a good idea to let `libosdp`'s version change independently from `libosdp-sys`'s.

Reply:
Only `libosdp-sys` crate is kept in sync with LibOSDP; the `libosdp` crate (which consumes `libosdp-sys`) already has its own versioning. Since LibOSDP itself kinda-sorta follows SemVer, we should be good.

Reply:
> `libosdp` crate already has its own versioning

I completely missed that. For some reason, I thought they were on the same version. 🤦‍♂️